testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksChangeDataFeedCompatibility.java

```diff
@@ -379,6 +379,68 @@ public void testMergeDeleteIntoTableWithCdfEnabled()
         }
     }
 
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_EXCLUDE_73, PROFILE_SPECIFIC_TESTS})
+    public void testMergeMixedDeleteAndUpdateIntoTableWithCdfEnabled()
+    {
+        String tableName1 = "test_merge_mixed_delete_and_update_into_table_with_cdf_" + randomNameSuffix();
```

Comment:
`tableName1` -> `targetTableName`
`tableName2` -> `sourceTableName`

as in the syntax of `MERGE` https://trino.io/docs/current/sql/merge.html